projects/step-core/src/lib/components/plan-tree/plan-tree.component.ts

```diff
@@ -20,32 +35,33 @@ import { PlanTreeAction } from '../../shared/plan-tree-action.enum';
   ],
 })
 export class PlanTreeComponent implements TreeActionsService {
+  private _treeState = inject<TreeStateService<AbstractArtefact, ArtefactTreeNode>>(TreeStateService);
+  private _planArtefactResolver? = inject(PlanArtefactResolverService, { optional: true });
+  readonly _planEditService = inject(PlanEditorService);
+  readonly _planInteractiveSession? = inject(PlanInteractiveSessionService, { optional: true });
+
   readonly selectedArtefact$ = this._treeState.selectedNode$.pipe(map((node) => node?.originalArtefact));
   @Input() isReadonly: boolean = false;
 
   @ViewChild('area') splitAreaElementRef?: ElementRef<HTMLElement>;
 
+  @ViewChild(TreeComponent) tree?: TreeComponent<ArtefactTreeNode>;
+
   private actions: TreeAction[] = [
+    { id: PlanTreeAction.OPEN, label: 'Open (Ctrl + O)' },
     { id: PlanTreeAction.RENAME, label: 'Rename (F2)' },
+    { id: PlanTreeAction.ENABLE, label: 'Enable (Ctrl + E)', hasSeparator: true },
+    { id: PlanTreeAction.DISABLE, label: 'Disable (Ctrl + E)', hasSeparator: true },
+    { id: PlanTreeAction.COPY, label: 'Copy (Ctrl + C)' },
+    { id: PlanTreeAction.PASTE, label: 'Paste (Ctrl + V)' },
+    { id: PlanTreeAction.DUPLICATE, label: 'Duplicate (Ctrl + D)' },
+    { id: PlanTreeAction.DELETE, label: 'Delete (Del)', hasSeparator: true },
```

Comment:
Why did you do changes to the plan-tree component?

Reply:
Ok it looks like something got messed up on the master merge, I will check this

Reply:
I fixed the issue. The problem happend since we merged master while the branch was pointing to the incorrect master

Reply:
ok, thank you @neogucky !